src/app/caupona-content/customers/customers.component.scss

```diff
@@ -0,0 +1,188 @@
+$main-color: #4bb1be;
+$main-color-dark: rgba(75, 177, 190, 0.81);
+$main-color-light: #a3c0dd;
+$main-color-dark-2: #00899b;
+$light-grey-2: #e1e1e1;
+$white: #fff;
+$dark-grey: #6d6d6d;
+$light-grey: #acacac;
+
+* {
+  font-family: "Reem kufi";
+}
+
+.customers {
+  display: grid;
+  grid-template-columns: 65% 1fr;
+  width: 97%;
+  grid-gap: 6em;
+  margin-left: auto;
+  position: relative;
+  height: calc(100vh - 70px - 2em);
+  grid-template-rows: calc(100vh - 70px - 2em);
+  > main {
+    padding: 2em;
+    margin-top: 5em;
+    background: #fff;
+    border: 1px solid rgb(240, 240, 240);
+    border-radius: 7px;
+    height: calc(100% - 5em);
+    > header.customers-header {
+      display: flex;
+      flex-flow: column;
+      height: 15%;
+      > span.customers-header__name {
+        font-size: 1.1em;
+        line-height: 1.65em;
+      }
+      > span.customers-header__activity {
+        font-size: 0.9em;
+        line-height: 1.35em;
+        color: $light-grey;
+      }
+    }
+    > section {
+      height: 85%;
+      display: flex;
+      flex-flow: column;
+
+      ul {
+        display: grid;
+        width: 100%;
+        justify-content: space-around;
+        margin: 0;
+        padding: 0em;
+        grid-template-columns: 1.5fr repeat(2, 1fr);
+        li {
+          text-align: center;
+          padding: 1em;
+        }
+      }
+
+      > header {
+        ul.visits-head {
+          background: #4bb1be;
+          border-radius: 5px 5px 0px 0;
+          padding-right: 6px;
+          color: #fff;
+          margin-top: 0.5em;
+          margin-left: 3em;
+          width: calc(100% - 3em);
+          li {
+            text-align: center;
+            border-right: 1px solid $light-grey-2;
+            &:last-of-type {
+              border-right: unset;
+            }
+          }
+        }
+      }
+      > article {
+        height: calc(100% - 15%);
+        overflow-y: scroll;
+        border: 1px solid $light-grey-2;
+        border-right: unset;
+        border-radius: 0 0 0 5px;
+        margin-left: 3em;
+        width: calc(100% - 3em);
+        ul.visits-body {
```

Comment:
jak piszesz w scss to staraj się nie korzystać ze zwykłego css'a tzn: 
`ul {
 &.visits-body {
...
}
}
`